src/game/objects/har.c

```diff
@@ -450,7 +451,7 @@ void har_floor_landing_effects(object *obj, bool play_sound) {
     int amount = rand_int(2) + 1;
     for(int i = 0; i < amount; i++) {
         int variance = rand_int(20) - 10;
-        vec2i coord = vec2i_create(obj->pos.x + variance + i * 10, obj->pos.y);
+        vec2i coord = vec2i_create(obj->pos.x / 256 + variance + i * 10, obj->pos.y / 256);
```

Comment:
Would it be better to use bitshift for faster divide ? pos.x >> 8 ?

Reply:
Though, compiler might already optimize this ? Maybe ?

Reply:
being explicit would probably help portability on platforms with... more simplistic... C compilers.

Reply:
Any compiler released in the last 30 years should do this.

Reply:
I prefer division over bitshifts, because bitshifts are more difficult for me to use and understand:
- i have to remember my left from my right
- i have to remember not to shift negative numbers
- i have to remember another operator's precedence